src/js/iotjs.js

```diff
@@ -203,8 +206,8 @@
       enumerable: false,
       get: function() {
         if (this.__stack__ === undefined) {
-          process._flushParserDumpFile();
-          this.__stack__ = makeStackTraceFromDump(this.__frames__ || []);
+          this.__stack__ = `Error: ${this.message}\n`
```

Comment:
The format of first line might be `${this.name}: ${this.message}\n` rather than a hardcoded `Error: xxx`.

Reply:
So an `TypeError` or any other `Error` inherited error class with it's own name can be printed as:
```
TypeError: unexpected string in arguments
   ...foobar stack trace
```

Reply:
Actually the `name` property of error instance is set by implementer manually rather than derived from the constructor's name.
yet LGTM🚀

Reply:
Maybe you forgot approve this Patch >.<